src/core/Pricing/data.tsx

```diff
@@ -267,11 +264,9 @@ export const consumptionData: PricingDataFeature[] = [
       color: "text-neutral-000",
       tooltip: (
         <p>
-          Clients establish and maintain a connection to the Ably service using
-          the most efficient transport available, typically WebSockets.{" "}
-          <a href="/docs/connect" className="ui-link">
-            Find out more.
-          </a>
+          We charge you for the amount of time devices are connected to our
+          network by minute. For example, if ten devices are each connected for
```

Comment:
Should this be "connected to our network by _THE_ minute"?

Reply:
I agree with you, but I've had this wording quoted to me both by the [v6 pricing copy sheet](https://docs.google.com/spreadsheets/d/1AQB7QhgfBrh8McPPPCGGBVvkNviq2Ct-q7KlqAGwdJU/edit?gid=1993871013#gid=1993871013), and directly in Slack, so either it's deliberate, or a propagated error from some point before that falls on saturated eyes.

There's also the mismatch of "ten" and "45" instead of picking one consistent way of expressing numbers that I disagree with, but I don't want to be on the end of a "why doesn't this copy" match feedback point later

Reply:
Good call on this, floated it and it was agreed to change the copy to this